deploy/ocm-agent-operator-managednotifications/10-managednotifications-cr.yaml

```diff
@@ -87,3 +87,9 @@ spec:
       severity: Error
       summary: "Filesystem has less than 3% inodes left"
       logType: Cluster Configuration
+    - activeBody: |-
+        Your cluster is currently alerting with 'KubeNodeUnschedulableSRE' because a worker node '${node}' can not drain. This is not an actionable alert for the SRE team. Please troubleshoot the node using this reference documentation: https://docs.openshift.com/rosa/rosa_cluster_admin/rosa-cordoning-nodes.
+      name: KubeNodeUnschedulableSRE
+      resendWait: 1
```

Comment:
Is 1 hour too soon to re-notify? I think as long as we send the first notification quickly that we could define this re-notification to be something like 6 hours, especially if they cordon'd the node on purpose for like a weekend - might not be great to send like 50 service logs about the same issue when they did it on purpose.  Open to other suggestions here, though :D

Reply:
Good point!